.github/workflows/test.yml

```
@@ -0,0 +1,34 @@
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Run tests
+        run: ./gradlew test
+
+      - name: Set test status
+        id: test_status
+        if: success()
+        run: echo "status=success" >> $GITHUB_OUTPUT
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)